frontend_tests/puppeteer_tests/08-admin.js

```diff
@@ -38,13 +38,16 @@ async function test_change_new_stream_notifications_setting(page) {
 
     const verona_in_dropdown =
         "#realm_notifications_stream_id_widget .dropdown-list-body > li:nth-of-type(1)";
+
+    await common.wait_for_text(page, verona_in_dropdown, "Verona");
     await page.waitForSelector(verona_in_dropdown, {visible: true});
-    await page.click(verona_in_dropdown);
+    await page.evaluate((selector) => $(selector).click(), verona_in_dropdown);
 
     await submit_notifications_stream_settings(page);
 
     const disable_stream_notifications =
         "#realm_notifications_stream_id_widget  .dropdown_list_reset_button";
+    await page.waitForSelector(disable_stream_notifications, {visible: true});
```

Comment:
I think you should do this change in a separate commit since this isn't part of the flake the first commit resolves. If it is part of the fix to the flake can you explain in the commit message.

Reply:
Made it a separate commit, thanks!